python/semantic_kernel/connectors/ai/ollama/services/ollama_chat_completion.py

```diff
@@ -30,11 +30,12 @@ class OllamaChatCompletion(TextCompletionClientBase, ChatCompletionClientBase, A
     Make sure to have the ollama service running either locally or remotely.
 
     Arguments:
-        ai_model_id {str} -- Ollama model name, see https://ollama.ai/library
+        ai_model_id {str} -- Ollama model name, defaults to llama2 (the most popular model at https://ollama.ai/library)
         url {Optional[Union[str, HttpUrl]]} -- URL of the Ollama server, defaults to http://localhost:11434/api/chat
         session {Optional[aiohttp.ClientSession]} -- Optional client session to use for requests.
     """
 
+    ai_model_id: str = "llama2"
```

Comment:
this is really not necessary, instantiating without a `ai_model_id` is not possible anyway, and model popularity is due to change over time, so I would rather not put this in. The only needed change to fix the issue is the one below.